.catalog-onboard-pipeline.yaml

```diff
@@ -6,7 +6,13 @@ offerings:
     catalog_id: 7df1e4ca-d54c-4fd0-82ce-3d13247308cd
     offering_id: 6d6ebc76-7bbd-42f5-8bc7-78f4fabd5944
     variations:
-      - name: standard
+      - name: security-enforced
+        mark_ready: true
+        install_type: fullstack
+        scc:
+          instance_id: 1c7d5f78-9262-44c3-b779-b28fe4d88c37
+          region: us-south
+      - name: fully-configurable
```

Comment:
Can you add this new feature to both variations?
https://github.com/terraform-ibm-modules/terraform-ibm-scc/blob/5a7ec37b7508dd05d8f9d86e6ef0ec301b058381/.catalog-onboard-pipeline.yaml#L15